src/nvim/eval.c

```diff
@@ -15140,8 +15140,11 @@ static void f_sockconnect(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     return;
   }
 
+  snprintf((char *)IObuff, sizeof(IObuff), "%s:%" PRIdLINENR,
```

Comment:
I intentionally did not want channel module digging around to get this info because (1) it's dependent on when the call is made, (2) the caller decides what "source" means (it may not always be a VimL function line:number), (3) channel is lower-level than eval module.